JobConfig/mixing/filters/DIOtail.fcl

```diff
@@ -0,0 +1,7 @@
+# remove steps descendant from stopping target DIOs generated in Mu2e code
+physics.producers.MuStopPileupFilter.selection: {
+  tool_type: ProcessVolumeDetectorStepAntiSelectionTool
+  processes: [ "mu2eMuonDecayAtRest" ]
+  volumes: [ "Foil_00", "Foil_01", "Foil_02", "Foil_03", "Foil_04", "Foil_05", "Foil_06", "Foil_07", "Foil_08", "Foil_09", "Foil_10", "Foil_11", "Foil_12", "Foil_13", "Foil_14", "Foil_15", "Foil_16", "Foil_17", "Foil_18", "Foil_19", "Foil_20", "Foil_21", "Foil_22", "Foil_23", "Foil_24", "Foil_25", "Foil_26", "Foil_27", "Foil_28", "Foil_29", "Foil_30", "Foil_31", "Foil_32", "Foil_33", "Foil_34", "Foil_35", "Foil_36" ]
```

Comment:
Will these foils be relevant for all target-stop processes? If so, I suggest to define this sequence in prolog, then you can source it wherever needed.

Reply:
That is likely; this sequence is now defined in `.../JobConfig/mixing/prolog.fcl`.